### ISSUE TITLE
Pyarrow is not able install in andriod studio 

### ISSUE THREAD

**Issue by deepik27** [chaquo/chaquopy#482]
I am getting when i use streamlit and streamlit is being installed but i am getting error in pyarrow 

 pip{
                install "wheel"
                install "opencv-python-headless"
                install "Pillow"
                install "opencv-python"
                install "pandas"
                install "numpy"
                install "matplotlib"
                install "scipy"
                install "scikit-learn"
                install "mpl-toolkits.clifford"
                install "mplot3d-dragger"
                install "more-itertools"
                install "random2"
                install "streamlit"
                install "dash"

            }

ERROR!!:

Failed to install pyarrow; python_version < "3.9" from https://files.pythonhosted.org/packages/62/d3/a482d8a4039bf931ed6388308f0cc0541d0cab46f0bbff7c897a74f1c576/pyarrow-3.0.0.tar.gz (from streamlit).

**Comment by mhsmith** [comment]
Thanks for the report. This package isn't currently a priority, but you can subscribe to this issue for any future updates. And if anyone else wants it too, please click the thumbs-up button above.

**Comment by deepik27** [comment]
Then what is the solution for this i have python file to display graph in streamlit that is plotly graph, i  am planing for paid license also the chaquopy....please give the solution for this

**Comment by mhsmith** [comment]
Sorry, I can't suggest any solution except to use something that doesn't require pyarrow.

**Comment by deepik27** [comment]
Can i use plotly in chaquopy to display graph

**Comment by mhsmith** [comment]
Let's discuss that in #483.

**Comment by deephudka05** [comment]
Is pyarrow supported now?

**Comment by mhsmith** [comment]
No, but the package build tool is now open-source, so if you'd like to try building it yourself, follow the instructions [here](https://github.com/chaquo/build-wheel). And if you're successful, please make a pull request so we can add the package to the public repository.

**Comment by black-born** [comment]
> No, but the package build tool is now open-source, so if you'd like to try building pyarrow yourself, follow the instructions [here](https://github.com/chaquo/build-wheel). And if you're successful, please make a pull request so we can add the package to the public repository.

I tried to build pyarrow but I run into this error and I don't know what should I do.
CMake Error at /home/jeremy/.local/lib/python3.8/site-packages/cmake/data/share/cmake-3.27/Modules/FindPackageHandleStandardArgs.cmake:230 (message):
  Could NOT find Python3 (missing: Python3_INCLUDE_DIRS
  Python3_NumPy_INCLUDE_DIRS Development.Module NumPy) (found version
  "3.8.10")
Call Stack (most recent call first):
  /home/jeremy/.local/lib/python3.8/site-packages/cmake/data/share/cmake-3.27/Modules/FindPackageHandleStandardArgs.cmake:600 (_FPHSA_FAILURE_MESSAGE)
  /home/jeremy/.local/lib/python3.8/site-packages/cmake/data/share/cmake-3.27/Modules/FindPython/Support.cmake:3824 (find_package_handle_standard_args)
  /home/jeremy/.local/lib/python3.8/site-packages/cmake/data/share/cmake-3.27/Modules/FindPython3.cmake:545 (include)
  cmake_modules/FindPython3Alt.cmake:51 (find_package)
  CMakeLists.txt:255 (find_package)

I used the command: `./build-wheel.py --python 3.8 --abi arm64-v8a packages/pyarrow`

here is my meta.yaml:
```
package:
  name: pyarrow
  version: "13.0.0"
  
requirements:
  build:
    - cmake
  host:
    - python
```

**Comment by mhsmith** [comment]
CMake's built-in Python support is quite complex, and has gone through a couple of different redesigns. Unfortunately I don't understand it well enough to give any help here. #790 may be related.

However, I have just pushed some changes to build-wheel which remove some of the special cases for CMake, making it just another [package installed from PyPI](https://pypi.org/project/cmake/#history). This should allow for more consistent builds, but it means the recipes will have to be updated to add a version number, like this:
```
    - cmake 3.28.1
```

Or if a package already has cmake in its pyproject.toml file, the line should simply be removed.
